digital-credentials/digital-credentials.tentative.https.html

```diff
@@ -1,122 +0,0 @@
-<!DOCTYPE html>
-<title>Digital Identity Credential tests.</title>
-<link rel="help" href="https://wicg.github.io/digital-identities/">
-<script src="/common/get-host-info.sub.js"></script>
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/resources/testdriver.js"></script>
-<script src="/resources/testdriver-vendor.js"></script>
-
-<body>
-<script type="module">
-import { buildValidNavigatorIdentityRequest, requestIdentityWithActivation } from './support/digital-identity-helper.js';
-
-// This regex removes the filename from the path so that we just get
-// the directory.
-const host = get_host_info();
-const basePath = window.location.pathname.replace(/\/[^\/]*$/, '/');
-const remoteBaseURL = host.HTTPS_REMOTE_ORIGIN + basePath;
-
-async function createIframeAndWaitForMessage(test, iframeUrl) {
-    const messageWatcher = new EventWatcher(test, window, "message");
-    var iframe = document.createElement("iframe");
-    iframe.src = iframeUrl;
-    document.body.appendChild(iframe);
-    const message = await messageWatcher.wait_for("message");
-    return message.data;
-}
-
-// Builds a valid navigator.identity.get() request where IdentityRequestProvider#request is an object.
-function buildValidNavigatorIdentityRequestWithRequestObject() {
-  return {
-      digital: {
-        providers: [{
-          protocol: "urn:openid.net:oid4vp",
-          request: {
-            // Based on https://github.com/openid/OpenID4VP/issues/125
-            client_id: "client.example.org",
-            client_id_scheme: "web-origin",
-            nonce: "n-0S6_WzA2Mj",
-            presentation_definition: {
-              // Presentation Exchange request, omitted for brevity
-            }
-          },
-        }],
-      },
-  };
-}
-
-// Requires browser to have mode where OS-presented digital-identity-prompt is
-// bypassed in favour of returning "fake_test_token" directly.
-promise_test(async t=>{
-  assert_false(navigator.userActivation.isActive);
-  let request = buildValidNavigatorIdentityRequest();
-  await promise_rejects_dom(t, "NetworkError", navigator.identity.get(request));
-}, "navigator.identity.get() fails if the page doesn't have user activation");
-
-promise_test(async t => {
-  let request = buildValidNavigatorIdentityRequest();
-  let credential = await requestIdentityWithActivation(test_driver, request);
-  assert_equals("urn:openid.net:oid4vp", credential.protocol);
-  assert_equals("fake_test_token", credential.data);
```

Comment:
It is a little unfortunate that we are removing this test, since it is what's helping us avoid regressions. Why does this need to be removed?

Reply:
These don't adhere to the spec. For example, "urn:openid.net:oid4vp" is not a thing that we have settled on and, per spec, it's impossible to get a credential without the user interacting with the credential selection sheet (hence this would never resolve). 

We need a separate WebDriver API that can vend credentials if we want something like this.

Obviously, the test would be fine an internal Chrome unit test, but it's not something that helps us here with interop. 

Make sense?

Reply:
> For example, "urn:openid.net:oid4vp" is not a thing that we have settled on 

The [`protocol`](https://wicg.github.io/digital-credentials/#the-providers-member) is a `DOMString` rather than an `enum`, so I wasn't thinking that we would necessarily need to settle on its value.

> We need a separate WebDriver API that can vend credentials if we want something like this.

I generally agree that we need a WebDriver API, but that is fairly involved. Any chance there is a way that we can keep this test without going all the way to a WebDriver API?

> Obviously, the test would be fine an internal Chrome unit test, but it's not something that helps us here with interop.

That's a reasonable compromise to me too: moving this specific test to a chrome-specific unit test as we improve the web platform tests infrastructure for the digital credential API.